packages/frontend/src/utils/account-with-lockup.js

```diff
@@ -92,16 +92,18 @@ export async function transferAllFromLockup(missingAmount) {
     }
 
     console.info('Attempting to transfer from lockup account ID:', lockupAccountId);
```

Comment:
this log should be inside of the if check so we don't inaccurately log that we're trying to transfer when we actually aren't due to there being a 0 liquid balance.